controllers/file.js

```diff
@@ -5,14 +5,14 @@ const express = require('express');
 const router = express.Router();
 const multer = require('multer');
 const AWS = require('aws-sdk');
-const request = require('request');
+const Model = require('hof').model;
```

Comment:
Not part of this work, but still don't think it's good practice to bring the whole hof model in just to use it for axios request

Reply:
Out of scope for this work. Please create a JIRA ticket to investigate